rom/laghos_utils.cpp

```diff
@@ -195,13 +195,13 @@ void split_line(const std::string &line, std::vector<std::string> &words)
 void SetWindowParameters(Array2D<int> const& twparam, ROM_Options & romOptions)
 {
     const int w = romOptions.window;
-    romOptions.dimX = twparam(w,0);
-    romOptions.dimV = twparam(w,1);
-    romOptions.dimE = twparam(w,2);
+    romOptions.dimX = min(romOptions.dimX, twparam(w,0));
```

Comment:
Since `romOptions` is passed by reference, this will change `dimX` for each window, which can reduce the maximum applied to subsequent windows. You may want to have a different variable to specify the maximum.

Reply:
@dylan-copeland Thanks for pointing that out. I will fix it.